lib/dalli/client.rb

```diff
@@ -353,9 +352,7 @@ def ring
             server_options[:password] = uri.password
             s = "#{uri.host}:#{uri.port}"
           end
-          options = @options.dup.merge(server_options)
-          options.delete(:digest_class)
-          Dalli::Server.new(s, options)
```

Comment:
just a cleanup... the `dup` is not necessary as `merge` creates a new Hash... happy to re-add the `:digest_class` key deletion if necessary, but `Dalli::Server` doesn't use this key